docs/release-notes.rst

```diff
@@ -6,10 +6,91 @@
  Release Notes
 ###############
 
+**************
+ Version 0.33
+**************
+
+Version 0.33.0
+==============
+
+**Release Date:** May 29, 2024
+
+**Breaking Changes**
+
+-  Helm: An entry for ``initialUserPassword`` is now required when running ``helm install``.
+   Existing deployments are unaffected. See :ref:`Helm Chart <helm-config-reference>`.
+
+-  Web UI: Enforce password requirements for all new non-remote users: see
+   :ref:`password-requirements`
+
+   -  This applies to users created using the ``Add User`` button in the Web UI for admins.
+
+   -  Admins may also change the passwords of other users with this same interface.
+
+   -  This does not affect logging in with an existing user who already has an empty or
+      non-compliant password, but we recommend setting good passwords for such users as soon as
+      possible.
+
+**Improvements**
+
+Kubernetes: Add Determined resource information such as ``workspace`` and ``task ID`` as pod labels.
+This improvement facilitates better resource tracking and management within Kubernetes environments.
+
+Configuration: Introduce a DCGM Helm chart and Prometheus configuration to the
+``tools/observability`` directory. Additionally, two new dashboards, "API Monitoring" and "Resource
+Utilization", have been added to improve observability and operational insight.
```

Comment:
Have been added to...where? Grafana right?

Reply:
Yeah after looking at the code change, they've been added to Grafana. I updated the wording to explicitly call that out.

Reply:
Oh unless you mean added to the tools/observability directory

Reply:
If so, I'm good, apologies I didn't read it correctly